website/templates/includes/sidenav.html

```diff
@@ -67,6 +67,14 @@
 		</ul>
 
 		
+		<ul class="space-y-3 font-medium text-3xl mt-8 mb-0">
+			<li class="mb-[-6px] ms-[1px]">
+				<span class="font-bold text-[#AAA] text-[1.2rem] tracking-wide">Feedback</span>
+			</li>
```

Comment:
Put this in another pr

Reply:
I have already made a pr on this please check I'll remove it from here